README.md

```diff
@@ -51,6 +51,16 @@ ParadeDB and its extensions can be deployed in one of two ways:
 
 For more information, including enterprise features and support, please [contact us by email](mailto:sales@paradedb.com).
 
+### Using the Setup Script
+
+For an interactive setup run the following in your terminal.
+
+```bash
+curl -O https://raw.githubusercontent.com/paradedb/paradedb/dev/deploy.sh && bash deploy.sh
```

Comment:
@destrex271 Could you please update this using the new syntax for CURLing from the website?